core/tee/tee_svc_cryp.c

```diff
@@ -1427,6 +1428,7 @@ static TEE_Result copy_in_attrs(struct user_ta_ctx *utc,
 	if (MUL_OVERFLOW(sizeof(struct utee_attribute), attr_count, &size))
 		return TEE_ERROR_OVERFLOW;
 
+	usr_attrs = memtag_strip_tag_const(usr_attrs);
```

Comment:
nitpicking: deserves an empty line below (IMHO)
ditto for the several other change functions below where such preamble is called prior using the passed userland memory references.